### ISSUE TITLE
Use "Completed Order" component in Profile page

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#3018]
We use the old completed order Blaze template in the profile page but we should reuse the Completed Order component that was converted to React and made marketplace-aware.

**Comment by janus-reith** [comment]
Maybe we could even take this a step further?
The orders are shown on the completed page, the profile page, the orders dashboard panel and the eMail template.
It would be great to generalize this somehow, to make customization easier.

**Comment by zenweasel** [comment]
@janus-reith I would agree except maybe for the orders dashboard as that's an entirely different view with different permissions (especially as it pertains to marketplace). But I think sharing this code for everywhere a user looks at their own order makes a lot of sense.

**Comment by rymorgan** [comment]
This hasn't been fully reviewed or approved, but that's my plan with the work I've done on our revamped user profile views. https://scene.zeplin.io/project/58af8900737c137280322bd6/screen/59b7272087fcafc969870826

**Comment by zenweasel** [comment]
@rymorgan Do I need to wait for any designs for the task in the title or can I have someone start on that?

**Comment by rymorgan** [comment]
@zenweasel You can start this now. As you can see from the zeplin designs, I want us to make the user profile pages more user-friendly in the near future. Those new designs assume we use the complete order template here as well so it'd be good to do it now so it's ready for the redesign.

**Comment by rymorgan** [comment]
@efalayi Yes, that's the point of this issue. Let's bring it into the profile as it is now (the new react component that Brent built). It'll probably need some design adjustments, but this should use the same component.

**Comment by spencern** [comment]
Resolved by #3112
